LocalWiki.php

```diff
@@ -178,13 +178,22 @@ function onInactiveSiteNoticeAfter( &$siteNotice, $skin ) {
 // Per-wiki settings
 if ( $wgDBname === 'erislywiki' ) {
 	$wgHooks['OutputPageParserOutput'][] = 'onOutputPageParserOutput';
-	function onOutputPageParserOutput( OutputPage &$out, ParserOutput $parseroutput ) {
+	function onOutputPageParserOutput( OutputPage &$out ) {
```

Comment:
Function should be renamed to for new hook

Reply:
Done.